spec/helpers/views_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+  describe ViewsHelper, type: :helper do
+    include ViewsHelper
+
+
+    describe "Creating an offender sar" do
+      context "adding a rejected offender sar case" do
+        describe Case, type: :model do
+
+          it "creates an offender object" do
+            # Use the factory to create an instance of Case::Sar::Offender
+            kase = FactoryBot.build(Case::Sar::Offender)
```

Comment:
Using `FactoryBot.build` is fine, but you can just use `build` instead as a shortcut.

There are many different factories defined in various files under spec/factories. These will set the attributes to use. It would probably be worth reading how FactoryBot works starting from https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#defining-factories